experimental/dds/tree2/src/feature-libraries/editable-tree-2/proxies/proxies.ts

```diff
@@ -151,7 +151,7 @@ export function createObjectProxy<TSchema extends StructSchema, TTypes extends A
 	content: TypedNodeUnion<TTypes>,
 	schema: TSchema,
 ): SharedTreeObject<TSchema> {
-	const nodeApi = createNodeApi(content);
+	const nodeApi = content;
 
```

Comment:
@CraigMacomber - If you look a little further done in the proxy implementation, you'll see there are two symbols being used:

[nodeSym] returns the 'nodeApi'.

[treeNodeSym] returns the 'content' as the "back door" to access the EditableTreeNode.

Now that 'nodeApi' and 'content' are the same object, do you think we should combine symbols?